docs/extra.js

```diff
@@ -0,0 +1,22 @@
+document$.subscribe(function () {
```

Comment:
issue: I tried this out and it seems to not work when I reload the page and keep the same scroll location. The header doesn't disappear until I scroll up and scroll down again.

However, removing the `document$.subscribe` and calling the function immediately seems to resolve the issue. My guess is that if the page is already scrolled down, then we never observe a mutation in the header for whatever reason. It looks like `extra.js` is embedded in the page after the target header element, so there should be a guarantee that the element can be addressed and we can install the mutation observer safely?

nit: Should this `function` also be an arrow function instead, like the other ones in this file?

Reply:
Ok thanks, I’ll try without subscribe then